cps/src/main/java/org/jenkinsci/plugins/workflow/cps/nodes/StepEndNode.java

```diff
@@ -38,6 +38,17 @@ protected String getTypeDisplayName() {
     }
 
     @Override
+    protected String getTypeFunctionName() {
+        StepDescriptor d = getDescriptor();
+        boolean isBody = getStartNode().isBody();
+        if (isBody) {
+            return "} //" + (d != null ? d.getFunctionName() : getStartNode().getStepName());
+        } else {
+            return getStartNode().getStepName() + " : End";
```

Comment:
No, it's [returning a class field directly](https://github.com/jenkinsci/workflow-plugin/blob/c2afa61d28b2922df955bf7f6b45e456e898c10c/api/src/main/java/org/jenkinsci/plugins/workflow/graph/BlockEndNode.java#L50).